src/pages/WalletProviders.tsx

```diff
@@ -0,0 +1,206 @@
+import {
+  Card,
+  Heading,
+  Icon,
+  Toggle,
+  Title,
+  Notification,
+  Modal,
+  Button,
+} from "@stellar/design-system";
+import { useDispatch } from "react-redux";
+import { InfoTooltip } from "components/InfoTooltip";
+import { SectionHeader } from "components/SectionHeader";
+import { useRedux } from "hooks/useRedux";
+import { useState, useEffect } from "react";
+import { AppDispatch } from "store";
+import { getWalletsAction, updateWalletAction } from "store/ducks/wallets";
+
+export const WalletProviders = () => {
+  const [updateWalletModal, setUpdateWalletModal] = useState({
+    visible: false,
+    walletId: "",
+    enabled: false,
+  });
+
+  const { wallets } = useRedux("wallets");
+  const dispatch: AppDispatch = useDispatch();
+
+  useEffect(() => {
+    if (!wallets.status) {
+      dispatch(getWalletsAction());
+    }
+  }, [wallets.status, dispatch]);
+
+  const myWallets = wallets?.items.filter((e) => e.enabled);
+  const avalaibleWallets = wallets?.items.filter((e) => !e.enabled);
+
+  const handleCloseModal = () => {
+    setUpdateWalletModal({ visible: false, walletId: "", enabled: false });
+  };
+
+  const handleUpdateWallet = async (walletId: string, enabled: boolean) => {
+    await dispatch(updateWalletAction({ walletId, enabled }));
+    window.location.reload();
```

Comment:
In general, I think we shouldn't reload the window to force the update. We'll need to fetch all necessary info that wasn't related to this again + UX isn't as smooth.

We can leave it for now because we'll update it to use TanStack Query, so there is no need to make it more complicated now with Redux.